defs/gcs_uploader.py

```diff
@@ -46,6 +46,6 @@ def main(argv):
 
     parser.add_argument("--manifest", required=True, help="path to manifest of targets")
     parser.add_argument("--root", required=True, help="path to root of workspace")
-    parser.add_argument("gcs_path", required=True, help="path in gcs to push targets")
+    parser.add_argument("gcs_path", help="path in gcs to push targets")
```

Comment:
should there be some sort of check that this is set? or will `gsutil` fail sufficiently on an empty path?